### DISCUSSION TITLE
Slash like commands to enable llm features.

### DISCUSSION THREAD

**Discussion by nikhilmehta16** [ether/etherpad-lite#5948] [Q&A]
Hello Everyone,
I am looking for an extensible text editor to add slash commands for features like Import from CRM and run a LLM summary on selected text. 
Is there an easy way to create a slash commands plugin and a little help on getting started?
If there is a way and with the help of some guidance, me and my co-founder would love to take this project up.

**Comment by SamTV12345** [comment]
Hi @nikhilmehta16 ,

I couldn't find a suitable plugin for your needs over on [the plugin viewer](https://static.etherpad.org/index.html) so you'd need to create a new plugin. There is this guide for developing a plugin: https://github.com/ether/etherpad-lite/wiki/Creating-a-plugin . You can also take a look at the other plugins over [here](https://github.com/ether).
